src/index.js

```diff
@@ -483,6 +483,9 @@ class SVGIcons2SVGFontStream extends Transform {
           (glyph.width - (bounds.maxX - bounds.minX)) / 2 - bounds.minX
         );
       }
+      if (this._options.centerVertically) {
+        glyphPath.translate(0, (fontHeight - glyph.height)/2)
```

Comment:
You should probably use the `const bounds = glyphPath.getBounds();` here like for the centerHorizontally option.

If your intent is to indeed have the ability to rely on the declared glyph height instead of its real path bounds, you could add an option like `useDeclaredGlyphSizes: h | v | hv` or something like that to release that behavior for both h/v centering or only one of them.

Reply:
Updated the PR to use get the height from the path bounds, as suggested.

I might create another PR with the `useDeclaredGlyphSizes` option later this week.